### ISSUE TITLE
Test accuracy is not similar with the paper

### ISSUE THREAD

**Issue by willyfh** [AlirezaShamsoshoara/Fire-Detection-UAV-Aerial-Image-Classification-Segmentation-UnmannedAerialVehicle#5]
Following are the results of my execution using your code (no modification in the hyperparameter and model):

model at 40 epoch,
(train) bin_accuracy: 0.9577
val_bin_accuracy: 0.9514
test_accuracy : 0.5724729895591736

model at 37 epoch (best model based on val set),
(train) bin_accuracy: 0.9568 
val_bin_accuracy: 0.9820
test_accuracy :  0.6100730895996094
confusion matrix:
[[ 814. 2666.]
[ 694. 4443.]]

train and validation accuracy is actually similar enough, but the test accuracy is quite far from the paper.

**Comment by AlirezaShamsoshoara** [comment]
It shouldn't be like this. I will check it and I will let you know.
